skimage/restoration/_denoise.py

```diff
@@ -99,9 +101,11 @@ def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
                              "``multichannel=True`` for 2-D RGB "
                              "images.".format(image.shape))
 
+    if win_size is None:
+        win_size = max(5, 2*int(ceil(3*sigma_spatial))+1)
```

Comment:
For a filter window in DSP, 5 is the lower limit where they have real meaning. It should be odd and at least 5.